.scalafmt.conf

```
@@ -1,16 +1,15 @@
+runner.dialect = scala213
 align.openParenCallSite = false
 align.openParenDefnSite = false
 continuationIndent.defnSite = 2
 danglingParentheses.preset = true
 docstrings.style = Asterisk
 importSelectors = singleLine
 maxColumn = 120
-verticalMultiline.newlineBeforeImplicitKW = true
 rewrite.redundantBraces.stringInterpolation = true
 rewrite.rules = [
   RedundantParens,
   PreferCurlyFors,
   SortImports
 ]
-unindentTopLevelOperators = false
-lineEndings=preserve
+lineEndings = preserve
```

`unindentTopLevelOperators` can be replaced with `indentOperator.topLevelOnly`. AFAIR it worked good enough in one project I was migrating in the past from scalafmt 2.7.5 -> 3.8.1 
```suggestion
indentOperator.topLevelOnly = false
lineEndings = preserve
```

---

Ah, thanks! Done